### PR TITLE
Turn off logging for containers for some classes

### PR DESCRIPTION
**Pull request by hmusum**
Make it possible to set logctl specs for all types of containers and set default logctl specs for some external classes that we want to avoid in vespa log

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java [hunk 1]
```diff
@@ -331,4 +325,25 @@ public boolean multitenant() {
...
     public ApplicationType getApplicationType() { return applicationType; }
 
+    public List<LogctlSpec> getLogctlSpecs() {
+        return logctlSpecs;
+    }
+    public void addLogctlCommand(String componentSpec, String levelsModSpec) {
+        logctlSpecs.add(new LogctlSpec(componentSpec,  levelsModSpec));
+    }
+
+    private static Set<LogctlSpec> setDefaultLogctlSpecs() {
```
Comment (arnej27959) [inline config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java:335]: consider renaming this method to "alwaysLogctlSpecs()" or just "defaultLogctlSpecs()"

File: config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainerCluster.java [hunk 2]
```diff
@@ -124,19 +124,12 @@ public ApplicationContainerCluster(TreeConfigProducer<?> parent, String configSu
...
-            }});
-    }
-
     @Override
     protected void doPrepare(DeployState deployState) {
         addAndSendApplicationBundles(deployState);
         sendUserConfiguredFiles(deployState);
         createEndpointList(deployState);
-        wireLogctlSpecs();
+        super.doPrepare(deployState);
```
Comment (arnej27959) [inline config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainerCluster.java:132]: super() should be first in method

**Review by arnej27959 — DISMISSED** [review]
looks good; my LogctlSpec seems clunky in retrospect, I will try to come up with a small improvement.

File: config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainerCluster.java (hunk 2 above)
Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainerCluster.java]: Fixed

File: config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java (hunk 1 above)
Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java]: Yup, that's better, fixed.

**Review by gjoranv — COMMENTED** [review]
👍

**Pull request merged**